docs/developer/permissions.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Permissions
+sidebar_label: Permissions
+---
+
+## User permissions
+
+Instead of assigning permissions directly to the user, we define them on a by-group basis. Organizing access rights into [Groups](developer/api-reference/objects/group.mdx) helps in determining staff users' roles in the organization.
+
+Examples of groups:
+- Translators - `MANAGE_TRANSLATIONS` permission.
+- Customer support - `MANAGE_ORDERS` and `MANAGE_USERS` permissions.
+
+### Creating and removing groups
+To create a new group, use [permissionGroupCreate](developer/api-reference/mutations/permission-group-create.mdx).
+
+Request:
+```graphql
+mutation {
+  permissionGroupCreate(
+    input: {
+      addPermissions: [MANAGE_GIFT_CARD, MANAGE_DISCOUNTS]
+      addUsers: []
+      name: "Sale managers"
+    }
+  ) {
+    errors {
+      message
+    }
+    group {
+      id
+      name
+      permissions {
+        name
+      }
+    }
+  }
+}
+```
+
+Successful response:
+```json
+{
+  "data": {
+    "permissionGroupCreate": {
+      "errors": [],
+      "group": {
+        "id": "R3JvdXA6NDM=",
+        "name": "Sale managers",
+        "permissions": [
+          {
+            "name": "Manage sales and vouchers.",
+          },
+          {
+            "name": "Manage gift cards.",
+          }
+        ]
+      },
+    }
+  }
+}
+```
+
+To remove group, use [permissionGroupDelete](developer/api-reference/mutations/permission-group-delete.mdx) mutation.
```

Comment:
IMO would be nice to have similar example for removing as we have for creating group

Reply:
Felt that adding an example for that doesn't add anything to the document besides making it longer.